science/projects/ketabi/AI_project/avatar_audio_to_video/sad_talker_quick_demo.ipynb

```diff
@@ -0,0 +1,1366 @@
+{
```

Comment:
<p>Great job, keep it up.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>